adapter/main.go

```diff
@@ -206,7 +206,17 @@ func main() {
 		return
 	}
 
-	kedaProvider, stopCh, err := cmd.makeProvider(ctx, time.Duration(globalHTTPTimeoutMS)*time.Millisecond)
+	controllerMaxReconciles := 1
+	controllerMaxReconcilesStr := os.Getenv("KEDA_METRICS_CTRL_MAX_RECONCILES")
+	if controllerMaxReconcilesStr != "" {
+		controllerMaxReconciles, err = strconv.Atoi(controllerMaxReconcilesStr)
+		if err != nil {
+			logger.Error(err, "Invalid KEDA_METRICS_CTRL_MAX_RECONCILES")
+			return
+		}
+	}
```

Comment:
```suggestion
	controllerMaxReconciles, err := strconv.Atoi(os.Getenv("KEDA_METRICS_CTRL_MAX_RECONCILES"))
        if err != nil {
		logger.Error(err, "Invalid KEDA_METRICS_CTRL_MAX_RECONCILES")
		return
	}
```

Reply:
Isn't this gonna fail if the ENV property is not being set?